lib/Capabilities/CapabilitiesDataType.js

```diff
@@ -0,0 +1,100 @@
+
+const CapabilitiesDataType = ({ content, readOnly, onChangeCapabilityCheckbox, isCapabilitySelected, isCapabilityDisabled }) => {
+  const { formatMessage } = useIntl();
+
+  const columnMapping = {
+    application: (
+      formatMessage(columnTranslations.application)
+    ),
+    resource: formatMessage(columnTranslations.resource),
+    view: formatMessage(columnTranslations.view),
+    edit: formatMessage(columnTranslations.edit),
+    create: formatMessage(columnTranslations.create),
+    delete: formatMessage(columnTranslations.delete),
+    manage: formatMessage(columnTranslations.manage),
+    // policies: formatMessage(columnTranslations.policies)
+  };
+
+  /**
+   * Renders an action checkbox for an item.
+   *
+   * @param {Object} item - The capability item object.
+   * @param {string} action - The action to render checkbox for action(e.g. 'view').
+   * @return {JSX.Element} By requirements on non-readonly mode we should hide the checkboxes on
+   * not related actions. Since every capability have only one action
+   * (e.g. {...capability, action: 'view'}),
+   * show only checkbox for that particular action.
+   * If readOnly mode we should show the checkbox for all actions.
+   */
+  const renderItemActionCheckbox = (item, action) => {
+    return <ItemActionCheckbox
+      action={action}
+      readOnly={readOnly}
+      isCapabilityDisabled={isCapabilityDisabled}
+      onChangeCapabilityCheckbox={onChangeCapabilityCheckbox}
+      item={item}
+      isCapabilitySelected={isCapabilitySelected}
+    />;
+  };
+
+  const resultsFormatter = {
+    application: (item) => item.applicationId,
+    resource: (item) => item.resource,
+    view: (item) => renderItemActionCheckbox(item, 'view'),
+    edit: (item) => renderItemActionCheckbox(item, 'edit'),
+    create: (item) => renderItemActionCheckbox(item, 'create'),
+    delete: (item) => renderItemActionCheckbox(item, 'delete'),
+    manage: (item) => renderItemActionCheckbox(item, 'manage'),
+    // policies: item => <Badge>{item.policiesCount}</Badge>
+  };
+
+  return (
+    <div data-testid="capabilities-data-type">
+      <Headline size="large" margin="none" tag="h3">
+        <FormattedMessage id="ui-authorization-roles.details.data" />
+      </Headline>
+      <MultiColumnList
+        interactive={false}
+        columnMapping={columnMapping}
+        formatter={resultsFormatter}
+        contentData={content}
+        visibleColumns={[
+          'application',
+          'resource',
+          'view',
+          'edit',
+          'create',
+          'delete',
+          'manage',
+          '',
+        ]}
+        columnWidths={{
+          application: '40%',
+          resource: '24%',
+          view: '6%',
+          edit: '6%',
+          create: '6%',
+          delete: '6%',
+          manage: '6%',
+        }}
+      />
+    </div>
+  );
+};
+
+CapabilitiesDataType.propTypes = { content: capabilitiesPropType,
```

Comment:
Please re-format propTypes in new components:
```
propTypes = {
...props
};
```

Reply:
Can you explain more ?

Reply:
I mean that there should be no properties on the line with open curly bracket
```
CapabilitiesDataType.propTypes = {
  content: capabilitiesPropType,
  ...
}
```

Reply:
done